src/test/java/com/epam/ld/module2/testing/MailServerTest.java

```diff
@@ -1,4 +1,33 @@
 package com.epam.ld.module2.testing;
 
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
 public class MailServerTest {
+
+    @Test
+    public void checkSuccessfulSendMessage() {
+        MailServer mailServer = new MailServer();
+        Boolean result  = mailServer.send("test@gmail.com", "just hi");
+        assertTrue(result);
+    }
+
+    @Test
+    public void checkUnSuccessfulSendingMessage() {
+        MailServer mailServer = new MailServer();
+        Boolean result  = mailServer.send("test@gmail.com", null);
+        assertFalse(result);
+    }
+
+    @Test
+    public void shouldThrowNullPointerExecution() throws NullPointerException {
+        MailServer mailServer = new MailServer();
+        try {

```

Comment:
There is `org.junit.jupiter.api.Assertions#assertThrows(java.lang.Class<T>, org.junit.jupiter.api.function.Executable)` specifically for such cases.